### ISSUE TITLE
`Buffer()` Deprecation Notice

### ISSUE THREAD

**Issue by loganpowell** [apex/up#783]
## Prerequisites

* [x] I am running the latest version. (`up upgrade`)
* [x] I searched to see if the issue already exists.
* [ ] I inspected the verbose debug output with the `-v, --verbose` flag.
* [x] Are you an Up Pro subscriber?

## Description

Getting this in AWS console:

```
ERROR (node:8) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods 
```

## Steps to Reproduce

I'm not sure this is coming from `Up`, but 

` node --trace-deprecation --pending-deprecation ...``` 

I don't know how to do this on Up directly

I did find this:

https://github.com/apex/up/blob/5f16b7f99d6c5bd23289e8bf1927a5cce32e3475/internal/shim/byline.js#L143

Here's the [reference](https://nodejs.org/fr/docs/guides/buffer-constructor-deprecation/)

>The Buffer() and new Buffer() constructors are not recommended for use due to security and usability concerns. Please use the new Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() construction methods instead.

**Comment by loganpowell** [comment]
This seems to be a popular polyfill: https://www.npmjs.com/package/safe-buffer

**Comment by tj** [comment]
boom, fixed in 7799044c by @skabbes :D
